open-redirect.yaml

```diff
@@ -0,0 +1,31 @@
+rules:
+- id: open-redirect
+  mode: taint
+  pattern-sources:
+  - patterns:
+    - pattern: $REDIRECTURL
+  pattern-sinks:
+  - patterns:
+    - pattern-inside: 'Redirect($REDIRECTURL)
```

Comment:
For a sink, you can use `- pattern: Redirect(...)`, you don't need the pattern inside or the `patterns`.